src/libraries/System.Data.OleDb/src/OleDbStruct.cs

```diff
@@ -33,11 +33,35 @@ internal enum DBBindStatus
     }
 #endif
 
-#if (WIN32 && !ARCH_arm)
     [StructLayoutAttribute(LayoutKind.Sequential, Pack = 2)]
-#else
-    [StructLayout(LayoutKind.Sequential, Pack = 8)]
+    internal struct tagDBPARAMBINDINFO_x86
+    {
+        internal IntPtr pwszDataSourceType;
+        internal IntPtr pwszName;
+        internal IntPtr ulParamSize;
+        internal int dwFlags;
+        internal byte bPrecision;
+        internal byte bScale;
+
+#if DEBUG
```

Comment:
If you keep this I suggest "#if 0" since one would only need it in DEBUG if specifically looking at this problem. It's not a big deal but it does override a member that in theory other code could call.

Reply:
I did this to keep the style in the rest of this file. Otherwise I don't mind just removing the whole thing